src/init/features/project.ts

```diff
@@ -98,6 +102,55 @@ async function selectProjectFromList(
   return toProjectInfo(project);
 }
 
+async function promptParentResource(): Promise<ProjectParentResource | undefined> {
```

Comment:
@mbleigh, @bkendall, @nbegley I'm adding an interactive process to let user specify parent resource when they want to create a new Firebase project.

Reply:
I thought at the design stage we discussed that we *wouldn't* make folder/org selection interactive since very few of our customers are likely to actually use it and it might be confusing / annoying.

Reply:
Sorry I should have explained more... The [projects:create](https://github.com/firebase/firebase-tools/blob/tc-management/src/commands/projects-create.ts#L55) command uses `--oraginzation` and `--folder`, but in this case we are in `firebase init` command whose main purpose is not to create projects, and moreover it's being used only in interactive mode. Adding the interactive flow as above seems to be the easiest way to make sure that everyone is able to create a new project. 

I also thought about a another solution here. Users can store their parent resource information in the config file (with configstore). I imagine about a new `firebase config <key> <value>` command here (which can be useful if we want to support further configurations in the future). This is more complex, but if you like it, I can write another proposal.

Reply:
I feel fairly strongly that if 99% of customers are never going to want it, we shouldn't prompt for it by default. What about just adding something to the text when going through the `init` project create flow that's something like:

> If you want to create a project in a Google Cloud organization or folder, please use firebase projects:create instead, and return to this command when you've created the project.

Reply:
Sounds good to me. Adding the text right after `Create a new project option` is chosen.

Reply:
Done